### DISCUSSION TITLE
Mention thinkbeforecoding article in the documentation?

### DISCUSSION THREAD

**Discussion by natalie-o-perret** [akhansari/EsBankAccount#9] [Ideas]
@akhansari Is it worthwhile to mention this awesome write-up https://thinkbeforecoding.com/post/2021/12/17/functional-event-sourcing-decider?

**Comment by akhansari** [comment]
💯 Yes I already did it by creating a link on the Decider word. Also referenced in the resources.

**Reply by natalie-o-perret** [reply]
Right, since the article was released ydy and haven't clicked on the new discussion as early as I thought, that might explain why I've missed it out.
